horace_core/sqw/@sqw/resolution_plot.m

```diff
@@ -190,42 +205,57 @@
 [inst, all_inst] = get_inst_class(w);
 if isempty(inst)
     if all_inst
-        error('The instrument type is not the same for all contributing raw data files')
+        error('HORACE:resolution_plot:invalid_argument',...
+            'The instrument type is not the same for all contributing raw data files')
```

Comment:
Same error on this if regardless of path.

Reply:
Should be OK. Its the confusing return from get_inst_class. Return variable inst will be non-empty only if both of the following are true (1) an instrument was defined for all contributing nxspe files in the sqw object and (2) all those instrumnets are of the same class. If inst is empty, then this can be because instrumnets are defined for all nxspe datasets (all_inst==true) but they are not all of the same class, or not all nxspe files had an instrument defined (all_inst==false)